ui/dashboards/src/components/EditJsonDialog/EditJsonDialog.tsx

```diff
@@ -18,18 +18,24 @@ import { useTimeRange } from '@perses-dev/plugin-system';
 import { useEditJsonDialog } from '../../context/DashboardProvider';
 import { useDashboard } from '../../context/useDashboard';
 
-export const EditJsonDialog = () => {
+export interface EditJsonDialogProps {
+  isCreating?: boolean;
```

Comment:
I think we could use a more generic term for the component: enableMetadataEdition or disableMetadataEdition

Reply:
Because isCreating is related to the user flow, not really the component